docs/en/uninstall.md

```diff
@@ -1,6 +1,6 @@
 To uninstall the AWS QRADD solution, you must delete the AWS CloudFormation stack. 
```

Comment:
for consistent, should remove AWS

I'm not sure how to make sure all occurrences are updated.